index.js

```diff
@@ -6,29 +6,97 @@ const db = require('./data/db.js');
 
 const server = express();
 
-// add your server code starting here
+server.use(express.json());
+const cors = require('cors')
+// server.use(cors());
 
-server.get('/', (req, res) =>{
+// add your server code starting here
+//get posts
+server.get('/api/posts',cors(), (req, res) =>{
     db.find().then(posts => {
-        res.status(418).json(posts);
+        res.status(200).json(posts);
     }).catch(err => {
-        res.json(err);
+        res.status(500).json({ error: "The posts information could not be retrieved." });
```

Comment:
Why have a named parameter (`err`) when you aren't using it? Might as well use an empty param list:
```javascript
.catch(() => { // ...
```